### ISSUE TITLE
[Bug report] Can not build with extended_image: ^8.1.0

### ISSUE THREAD

**Issue by tuandungnguyen** [fluttercandies/extended_image#608]
### Version

8.1.0 

### Platforms

dart

### Device Model

Galaxy Note 20

### flutter info

```sh
[√] Flutter (Channel stable, 3.13.0, on Microsoft Windows [Version 10.0.22621.2134], locale en-US)
    • Flutter version 3.13.0 on channel stable at D:\Android\flutter_sdk\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision efbf63d9c6 (9 days ago), 2023-08-15 21:05:06 -0500
    • Engine revision 1ac611c64e
    • Dart version 3.1.0
    • DevTools version 2.25.0
```

### How to reproduce?

extended_image: ^8.1.0

![image](https://github.com/fluttercandies/extended_image/assets/17266479/4a3c7dad-adf1-4c1a-9ebe-56731f2af67a)

There is a new issue after upgrading to v8.1.0
 extended_image: ^8.1.0
 
 Please support 

### Logs

```sh
flutter pub get               
Resolving dependencies... (1.6s)
Because extended_image >=8.1.0 depends on extended_image_library ^3.6.0 which depends on http_client_helper ^3.0.0, extended_image >=8.1.0 requires http_client_helper ^3.0.0.
And because http_client_helper >=3.0.0 depends on http ^1.0.0, extended_image >=8.1.0 requires http ^1.0.0.
So, because iplayer depends on both http ^0.13.6 and extended_image ^8.1.0, version solving failed.


You can try the following suggestion to make the pubspec resolve:
* Consider downgrading your constraint on extended_image: flutter pub add extended_image:^8.0.2
```

**Comment by zmtzawqlp** [comment]
```
dependency_overrides:
    http: xxx
```

**Comment by ameenbadri7** [comment]
cannot build on Channel stable, 3.13.1 also .. 
fvm flutter doctor                                                                                         
Doctor summary (to see all details, run flutter doctor -v):
[!] Flutter (Channel stable, 3.13.1, on macOS 13.5 22G74 darwin-arm64, locale en-YE)
    ! Warning: `flutter` on your path resolves to /Users/ameen/fvm/versions/3.10.6/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/ameen/fvm/versions/3.13.1. Consider adding /Users/ameen/fvm/versions/3.13.1/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/ameen/fvm/versions/3.10.6/bin/dart, which is not inside your current Flutter SDK checkout at /Users/ameen/fvm/versions/3.13.1. Consider adding /Users/ameen/fvm/versions/3.13.1/bin to the front of your path.
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[!] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    ✗ Flutter requires Xcode 14 or higher.
      Download the latest version or update via the Mac App Store.
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.81.1)
[✓] VS Code (version 1.58.0)
[✓] Connected device (2 available)
